docs/cpp_client/cpp_quick_start.md

```diff
@@ -20,52 +17,209 @@ Ubuntu 18.04.
 - Windows 10
 - [Visual Studio 2019 Community Edition](https://visualstudio.microsoft.com/vs/community/)
 - [VSCode](https://code.visualstudio.com/)
-- [Boost 1.73.0](https://sourceforge.net/projects/boost/files/boost-binaries/1.73.0/boost_1_73_0-msvc-14.2-64.exe/download)
-    *Note: Use the default install location*
+
+### Windows Library Dependencies 
+**NOTE**: Extract or install these libraries to `C:\local` so cmake can find them.
+
+- [Boost](https://sourceforge.net/projects/boost/files/boost-binaries/1.73.0/boost_1_73_0-msvc-14.2-64.exe/download)
+  - Add `C:\local\boost_1_73_0\lib64-msvc-14.2` to your PATH variable
+  - Create the `BOOST_ROOT` environment variable and set it to `C:\local\boost_1_73_0\`
+
+<div class="img_container">
+    <img class="lg_img" src="../imgs/boost_root_system_var.jpeg">
+</div>
+
+- [CMake](https://cmake.org/download/)
+    - When installing, make sure to choose the option to CMake to your Windows Path variable.
  
+To build the examples the following are required:
+
+- [OpenCV](https://github.com/opencv/opencv/releases/download/4.3.0/opencv-4.3.0-vc14_vc15.exe) Extract to `C:\local\opencv` and add `C:\local\opencv\build\x64\vc15\bin` to your PATH environment variable.
+
+- [Eigen](https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.zip) Extract to `C:\local\Eigen3` and add `C:\local\Eigen3` to your PATH environment variable.
+     - **NOTE**: You will need to move the extracted contents out of the version subfolder (eg `.\eigen-3.3.7\`) into the parent directory, such that `C:\local\Eigen3\Eigen` is a valid directory path.
+
+<div class="img_container">
+    <img class="lg_img" src="../imgs/eigen3_dir.jpeg">
+</div>
+
+#### Figure: Environment Variables
```

Comment:
For some reason using 4 `#` renders as a top level header on my browser. Suggest re-styling this: 
![image](https://user-images.githubusercontent.com/6026642/130680720-63d43316-95a5-418b-83f0-1c9c314980ff.png)